package.json

```diff
@@ -158,7 +158,9 @@
     "**/json5": "^2.2.3",
     "**/mime": "^3.0.0",
     "**/performance-now": "^2.1.0",
-    "**/prismjs": "^1.30.0"
+    "**/prismjs": "^1.30.0",
+    "**/webpack-dev-middleware": "^5.3.4",
```

Comment:
this i believe will cause storybook to break. have you tried `yarn storybook explore`?